### PR TITLE
Add quote

### PR DESCRIPTION
**Pull request by patrickbower**
Thought it might need a broad statement to encapsulate what C7 is about

### CODE REVIEW

File: src/components/quote.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const Quote = () => (
+  <section className="quote">
+    <div className="grav-o-full-bleed__content">
+      <p className="grav-u-font-size-plus-3">
+        "Using Cycle 7 we can measure the desirability, viability and feasibility for ideas without ambiguity."
```
Comment (danbull) [inline src/components/quote.js]: Should it be "of ideas"?

File: src/styles/_quote.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  margin: 170px 0 100px;
+
+  @media (max-width: grav-breakpoint(large)) {
+    margin: 120px 0 70px;
+  }
+
+  @media (max-width: grav-breakpoint(medium)) {
+    margin: 70px 0 50px;
+  }
+}
```
Comment (danbull) [inline src/styles/_quote.scss]: Needs replacing with Gravity vars

Comment (patrickbower) [reply src/styles/_quote.scss]: What _needs replacing with Gravity vars_ ?

Comment (danbull) [reply src/styles/_quote.scss]: I was mainly talking about the `color` but possibly the margins (multiples of $grav-sp-xl maybe). Not a biggie for now though.

**Pull request merged**